devops/tasks.py

```diff
@@ -281,13 +282,20 @@ def _should_ignore(path):
         logger.info(f"Keeping temporary kube merges in {merge_tmp}")
 
 
-def get_master_key(env: str) -> None:
+def get_master_key(env: str, use_existing=False) -> Path:
```

Comment:
I get you might want this to be `False` if you specifically issue a command to download the new one from CLI, but in other cases why would you default to `False`?

Reply:
Really good point. Changed it the other way around.